### ISSUE TITLE
Runas the Shamed

### ISSUE THREAD

**Issue by MatticusGib** [IllInuz/FelstormBT#42]
### Type bug

Quests Broken Isles

### Description

Runas the Shamed is an escort quest that begins in a cave, and ends by approaching a Projection of Senegos.  At the end, it doesn't automatically complete by getting close to Senegos, I even had to run around in circles and through the image several times before it triggered.

### Expected behaviour

It should be smoother to complete.

### Steps to reproduce the problem

You'll have to do several quests ahead of this one, starting with to Azure Repose after completing the first set of quests in Aszuna.

Basically, get to this quest, and try doing it, it should give you the same problem.

### Faction

Alliance

### Level, Class, and Spec

110 Human Brewmaster Monk

### Operating system

Windows 10 Pro

### Additional Notes

![WoWScrnShot_110923_123213](https://github.com/IllInuz/FelstormBT/assets/148009310/3db3c5fc-17bf-4429-9dd8-107b8cf146b9)
https://www.wowhead.com/quest=37957/runas-the-shamed
